code/go/0chain.net/smartcontract/dbs/event/blobber.go

```diff
@@ -121,6 +121,16 @@ func (edb *EventDb) addOrOverwriteBlobber(blobber Blobber) error {
 	return result.Error
 }
 
+func (bl *Blobber) existsV2(edb *EventDb) (bool, error) {
+	var exists bool
+	result := edb.Get().Raw("select exists(select 1 from blobbers where blobber_id = ? limit 1) as ex", bl.BlobberID).Scan(&exists)
```

Comment:
You can use .First query of gorm. Exactly same as GetBlobber
It's an extremely powerful query which will always order by on primary key and adds limit 1.

Reply:
Done